[DOCS-4815] Add clarification to log message remapper (#16999)
DataDog/documentation: content/en/logs/log_configuration/processors.md

## Log message remapper

> Message is a key attribute in Datadog. It is displayed in the message column of the Log Explorer to give context to a log. You can use the search to find a log by the log message. 

Author: ```suggestion
`message` is a key attribute in Datadog. Its value is displayed in the **Content** column of the Log Explorer to provide context on the log. You can use the search to find a log by the log message. 
```

> `message` is a key attribute in Datadog. Its value is displayed in the **Content** column of the Log Explorer to provide context on the log. You can use the search to find a log by the log message. 

Reviewer: ```suggestion
`message` is a key attribute in Datadog. Its value is displayed in the **Content** column of the Log Explorer to provide context on the log. You can use the search bar to find a log by the log message. 
```

> Use the log message remapper processor to define one or more attributes as the official log message. Define more than one attribute for cases when the attributes might not exist and an alternative is available. For example, if the defined message attributes are `attribute1`, `attribute2`, and `attribute3`, and `attribute1` does not exist, then `attribute2` is used. Similarly, if `attribute2` does not exist, then `attribute3` is used.

Author: ```suggestion
Use the log message remapper processor to define one or more attributes as the official log message. Define more than one attribute for cases where the attributes might not exist and an alternative is available. For example, if the defined message attributes are `attribute1`, `attribute2`, and `attribute3`, and `attribute1` does not exist, then `attribute2` is used. Similarly, if `attribute2` does not exist, then `attribute3` is used.
```
